core/src/interactive.rs

```diff
@@ -253,6 +257,7 @@ pub enum Message {
     SetChatLink(Option<String>),
     SetNumDecks(Option<usize>),
     SetKittySize(Option<usize>),
+    SetFriendSelectionPolicy(String),
```

Comment:
You can take a `FriendSelectionPolicy` here and not deal with manual serde

Reply:
done